lib/storage/base.ts

```diff
@@ -22,35 +22,36 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import profanities from 'profanities';
+import * as express from 'express';
 
 import {logger} from '../logger';
+import {CompilerProps} from '../properties';
 import * as utils from '../utils';
 
+// When it's import profanities from 'profanities'; ts says "Cannot find module 'profanities' or its corresponding type
+// declarations."
+// Updating profanities to v3 requires ESM modules
+// eslint-disable-next-line @typescript-eslint/no-var-requires, unicorn/prefer-module
+const profanities = require('profanities');
```

Comment:
I'm not sure if there's another way to do this. I could not get any `import` working.

Reply:
profanities is another lib we can't upgrade because of ESM, so... one day this will probably be ok.